multinet/multinet/__init__.py

```diff
@@ -31,6 +31,23 @@ def graphql_query(query, variables=None):
     return dict(data=result.data, errors=errors, query=query)
 
 
+def csv_validation(rows):
```

Comment:
Let's rename this to `validate_csv` so that the function name is a verb.

Reply:
Sure thing